### DISCUSSION TITLE
Filament sensor always starts disabled

### DISCUSSION THREAD

**Discussion by RoisinDubh79** [bassamanator/Sovol-SV06-firmware#119] [General]
Figured I'd go the discussion route on this.

I have a functional filament runout sensor installed and working.

I've modified the misc-macros config file to say
variable_filament_sensor_enabled 1

Every time I start klipper, my filament sensor is disabled and I need to enable it each time either via the dashboard switch or the ADJUST_FILAMENT_SENSOR_STATUS macro.

I noticed in misc-macros, there is this delayed gcode macro:
[delayed_gcode DISABLEFILAMENTSENSOR]

Does this run every time klipper starts?

**Comment by bassamanator** [comment]
Correct, as it should.

Test it out by removing the filament from the sensor after a print starts...

EDIT do not change the config, but make sure the sensor is enabled: `variable_filament_sensor_enabled 1`.

**Reply by RoisinDubh79** [reply]
The sensor is enabled using
variable_filament_sensor_enabled 1
but it always starts up disabled.
